README.md

```diff
@@ -108,12 +115,13 @@ GET `/`
 | Update an Address| PUT `/customers/{int:customer_id}/addresses/{int:address_id}`  
 | Delete an Address| DELETE `/customers/{int:customer_id}/addresses/{int:address_id}`           
 | List Addresses    | GET `/customers/{int:customer_id}/addresses`  
+| Search Customers and Addresses | GET `/customers/<query_field>=<query_value>`
```

Comment:
`/customers/?<query_field>=<query_value>`
Please put a question mark before the query field

Reply:
oops sorry will do it